source/layouts/layout.erb

```diff
@@ -56,7 +56,10 @@ under the License.
       <% end %>
       <% if current_page.data.search %>
         <div class="search">
-          <input type="text" class="search" id="input-search" placeholder="Search">
+          <form>
+            <input type="text" class="search" id="input-search" placeholder="Search">
+            <button type="reset" class="reset-form"><i class="fa fa-times"></i></button>
+          </form>
```

Comment:
No, there's not. I didn't go back and remove the html5 reference later. I have confirmed that JS is sufficient.